spec/helpers/country_helper_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe CountryHelper, type: :helper do
+  describe "#country_names_from_code_list" do
+    context "when there are countries passed in" do
+      it "returns the list of names for the codes of the countries passed in" do
+        activity = build(:project_activity, benefitting_countries: ["AR", "EC", "BR"])
+        result = country_names_from_code_list(activity.benefitting_countries, BenefittingCountry)
+        expect(result).to eql(["Argentina", "Ecuador", "Brazil"])
```

Comment:
`eq` for consistency? Likewise elsewhere in this file and `spec/presenters/activity_csv_presenter_spec.rb`